Sudoku/constants.js

```diff
@@ -0,0 +1,7 @@
+const SQUARES = require('./locationMap.js')
+const BOARD_LENGTH = 81
+const NUMBERS_PER_SQUARE = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+
```

Comment:
This is an interesting strategy.  Do the specs ask you to put constants in their own file like this?